wordle-web/pages/index.vue

```diff
@@ -0,0 +1,56 @@
+<template>
```

Comment:
2. Create a simple page called game.vue ❌
We must have game.vue file so you can just rename this file.

- and a button to take you back to the home page ❌
- there should be a button on the home page to access this page ❌
I can't really tell if your implementation is right or wrong because you have a home page that contains a game card. What I understand from this assignment is that we must have a home page by itself with a button to the game page. On that game page, we must have a game card placeholder and a button that leads to the home page.

Reply:
I second this, make sure to move your game into another page called `game.vue`(this could just replace `test.vue` I believe) and also make sure to add a button to go back and forth between the pages